page.php

```diff
@@ -1,5 +1,32 @@
 <?php
 get_header();
+
+function render_submenu($post) {
```

Comment:
says who? why exactly? functional code that is reused goes in functions.php so u can call from wherever. this is only reusable on this template so whats the point?